### PR TITLE
Fix UvDriver

### LINKED ISSUE #45: Broken ext-uv 0.2 support in dev-master
I wanted to take a look at the problem from one of my [PRs](https://github.com/amphp/file/pull/38#discussion_r418724296) and found out that the tests are not passing in dev-master when ext-uv v0.2 is installed. By checking out some older commits I found out that it was broken by the upgrade to ext-uv 0.3: https://github.com/amphp/file/commit/ae97b95e3e2bedb24831d317b38747f74d39a9fa.

### PR DESCRIPTION
**Pull request by enumag**
Closes #45

### CODE REVIEW

File: src/UvDriver.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function __construct(Loop\UvDriver $driver)
         $this->loop = $driver->getHandle();
         $this->poll = new Internal\UvPoll;
 
-        $this->priorVersion = \version_compare('0.3.0', \phpversion('uv')) < 0;
+        $this->priorVersion = \version_compare('0.3.0', \phpversion('uv'), '>');
```
Comment (kelunik) [inline src/UvDriver.php:44]:
```suggestion
        $this->priorVersion = \version_compare(\phpversion('uv'), '0.3.0', '<');
```

File: src/UvFile.php [hunk 2]
```diff
@@ -68,7 +68,7 @@ public function __construct(Loop\UvDriver $driver, Internal\UvPoll $poll, $fh, s
 
         $this->queue = new \SplQueue;
 
-        $this->priorVersion = \version_compare('0.3.0', \phpversion('uv')) < 0;
+        $this->priorVersion = \version_compare('0.3.0', \phpversion('uv'), '>');
```
Comment (kelunik) [inline src/UvFile.php:71]:
```suggestion
        $this->priorVersion = \version_compare(\phpversion('uv'), '0.3.0', '<');
```

File: test/UvDriverTest.php [hunk 3]
```diff
@@ -21,4 +21,18 @@ protected function setUp(): void
...
     }
+
+    /**
+     * @dataProvider readlinkPathProvider
+     *
+     * @param \Closure $linkResolver
+     */
+    public function testReadlinkError(\Closure $linkResolver): \Generator
+    {
+        if (\version_compare('0.3.0', \phpversion('uv'), '>')) {
```
Comment (kelunik) [inline test/UvDriverTest.php:32]:
```suggestion
        if (\version_compare(\phpversion('uv'), '0.3.0', '<')) {
```

**Comment by enumag** [comment]
@kelunik Suggestions applied.

**Comment by kelunik** [comment]
Thanks!

**Pull request merged**